### DISCUSSION TITLE
Github Desktop not working

### DISCUSSION THREAD

**Discussion by JohnnyBonk** [community/community#145700] [New to GitHub]
Hello,
I've tried reinstalling github desktop twice and logging and and out repreatedly and I still cannot manage to get any repo to open via desktop when im trying to download from github online. any help is greatly appreciated.

**Comment by ujamX** [comment]
Try using a different browser, clearing your cache and cookies, or using incognito mode.

If this doesn't work, please leave a reply.

**Reply by ghostinhershell** [reply]
Hi @BinkyTinky, we're happy you're here! You are more likely to get a useful response if you are explicit about the issue you're running into here, giving a few more details might help someone give you a nudge in the right direction. For example, did @ujamX's suggestions here help?

**Comment by Barmanji** [comment]
Could be a problem with installation.

Also Ensure GitHub Desktop is set as the default app for opening .git repositories.
On **Windows:**

    Right-click any .git file and select Open With > Choose Another App.
    Select GitHub Desktop and check Always use this app.

On **macOS:**

    Right-click and select Get Info.
    Under Open With, choose GitHub Desktop and click Change All.
    
Or-

GitHub Desktop uses protocol handlers to link with GitHub online. If they’re broken:

    On Windows:
        Open the Settings app, go to Apps > Default Apps > Choose Default by Protocol, and ensure x-github-client and github-mac:// point to GitHub Desktop.
    On macOS:
        Reinstall GitHub Desktop to reset these handlers automatically.
